### ISSUE TITLE
Bondwire.set_start_elevation()

### ISSUE THREAD

**Issue by svandenb-dev** [ansys/pyedb-core#450]
### 🔍 Before submitting the issue

- [X] I have searched among the existing issues
- [X] I am using a Python virtual environment

### 🐞 Description of the bug

set_start_elevation requires CellInstance and layer as argument. However CellInstance is optional, EDB .NET accept the current opened Cell as argument.

When I try to change the bondwire elevation I got this error.

![image](https://github.com/user-attachments/assets/46ae594f-a0eb-4893-9136-1eaff3415d53)

![image](https://github.com/user-attachments/assets/dbcef050-96f1-411f-963d-7c87d5130cbb)

![image](https://github.com/user-attachments/assets/11a288c0-6981-4375-a854-5097a154cad0)

### 📝 Steps to reproduce

NA

### 💻 Which operating system are you using?

Windows

### 🐍 Which Python version are you using?

3.11

### 📦 Installed packages

```shell
NA
```

**Comment by drewm102** [comment]
@svandenb-dev I don't think that passing the active `Cell` object is an option for this API in the .NET API. In the .NET API, the context parameter must either be a `CellInstance` or `null`. In the edb rpc API, it must be either a `CellInstance` or `None`. Can you try passing `None`? I think that should resolve this issue.

**Comment by drewm102** [comment]
@svandenb-dev I'm closing this out because of my previous comment. Feel free to reopen it in the future if needed though

**Comment by svandenb-dev** [comment]
[like]  Simon Vandenbrouck reacted to your message:
________________________________
